service/src/main/java/uk/gov/hmcts/reform/fpl/utils/PeopleInCaseHelper.java

```diff
@@ -32,4 +36,18 @@ public static String getFirstRespondentLastName(List<Element<Respondent>> respon
                 representative.getAddress().getAddressAsString(", ")))
             .collect(toList());
     }
+
+    public static String getAllocatedJudgeTitle(CaseData data) {
```

Comment:
Pass in `Judge` object rather than `CaseData`

Reply:
will do, good spot. :)